views/delete-job-role.html

```diff
@@ -0,0 +1,25 @@
+{% extends "templates/master.html" %}
```

Comment:
Can I have some screenshots of the UI please

Reply:
<img width="1306" alt="Screenshot 2023-09-21 at 12 10 03" src="https://github.com/anthony-raybould/bham1-front-end/assets/142424155/b3af4376-3
<img width="1274" alt="Screenshot 2023-09-21 at 12 10 40" src="https://github.com/anthony-raybould/bham1-front-end/assets/14242415
<img width="1217" alt="Screenshot 2023-09-21 at 12 10 54" src="https://github.com/anthony-raybould/bham1-front-end/assets/142424155/51aa359f-357c-4a7c-b9ee-6cc7737ffc32">
5/fb7bb610-fffb-4c8c-bce5-5a17663ad53c">
c69-4f34-8693-233fb7e9cd68">

Reply:
<img width="1306" alt="Screenshot 2023-09-21 at 12 10 03" src="https://github.com/anthony-raybould/bham1-front-end/assets/142424155/e2ab8177-653b-4cb2-baed-57c159e7f937">

Reply:
![Uploading Screenshot 2023-09-21 at 12.10.40.png…]()

Reply:
<img width="1274" alt="Screenshot 2023-09-21 at 12 10 40" src="https://github.com/anthony-raybould/bham1-front-end/assets/142424155/fb52e246-c6dc-4aff-898e-77bf5578ee25">

Reply:
<img width="1217" alt="Screenshot 2023-09-21 at 12 10 54" src="https://github.com/anthony-raybould/bham1-front-end/assets/142424155/30dbe61b-b1eb-4126-9193-ea0d1a8ece37">